README.md

```diff
@@ -133,6 +133,67 @@ enum Shape {
 }
 ```
 
+## Switch Statements
+1. Enum cases with an associated value should have a appropriate label as opposed to just types
+
+***Preferred:***
+
+```swift
+enum Trade {
+    case buy(stock: String, amount: Int)
+    case sell(stock: String, amount: Int)
+}
+```
+
+***Not Preferred:***
+
+```swift
+enum Trade {
+    case buy(String, Int)
+    case sell(String, Int)
+}
+```
+
+
+2. When using a switch statement that has a finite set of possibilities, do not include a _default_ case. Instead, place unused cases at the bottom and use the break keyword to prevent execution.
+
+```swift
+enum UserState {
+    case loggedIn
+    case loggedOut
+    case openCatalogue
+    case onboarding
+}
+```
+
+***Preferred:***
+
+```swift
+func handleUser(with state: UserState) {
```

Comment:
Not 100% agreed with this. I think it depends on the enum specially if the other options are completely irrelevant to what the switch is trying to address. Another example would be you have an error enum with multiple errors that and you can only act on a handful it becomes boilerplate to enumerate all of them just to follow a rule. For example something that I'm working for a personal project can throw this error https://developer.apple.com/documentation/networkextension/nehotspotconfigurationerror.
Of course this are different opinions and as usual there is a balance to achieve and preferences involved so if the rest of the team is ok with this I'll adhere

Reply:
Our own `NetworkServiceError` is another example.

Reply:
Perhaps this recommendation could be relaxed a bit?

There is a lot of value in not using the `default` case, but there are as @pablocaif suggests, instances in which it would be too cumbersome to do.